features/support/assemblies/dom-http-memory.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs')
+const path = require('path')
+const DomTodoList = require('../../../test_support/DomTodoList')
+const MemoryTodoList = require('../../../test_support/MemoryTodoList')
+const HttpTodoList = require('../../../lib/client/HttpTodoList')
+const BrowserApp = require('../../../lib/client/BrowserApp')
+const WebApp = require('../../../lib/server/WebApp')
+
+module.exports = class DomHttpMemoryAssembly {
+  constructor () {
+    this.memoryTodoList = new MemoryTodoList()
+    this.webApp = new WebApp({ todoList: this.memoryTodoList, serveClient: false })
+  }
+
+  async contextTodoList() {
+    return this.memoryTodoList
+  }
+
+  async actionTodoList() {
+    return await this.makeDomTodoList()
+  }
+
+  async outcomeTodoList() {
+    return await this.makeDomTodoList()
+  }
+
+  async makeDomTodoList() {
```

Comment:
I like to use `_` as a prefix for private methods, to make it apparent they're not part of the public API/contract.

Reply:
👍